### DISCUSSION TITLE
Screenshots of UI-scale issues

### DISCUSSION THREAD

**Discussion by Duke-M-commits** [ReactiveDrop/reactivedrop_translations#497] [General]
Language: English
Where: Play Online -> Create Game
Possible solution: renaming 'rd_campaign_name_rd-tarnorcampaign1' from `Tears For Tarnor, Ch.1: Insertion` into `Tears For Tarnor`

![image](https://user-images.githubusercontent.com/69652463/197586325-102bfcf1-f09b-47f2-834d-f06565b0a5e5.png)

**Comment by Duke-M-commits** [comment]
Language: German
Where: Options -> Advanced settings
Possible solution: widen the container of the label

![image](https://user-images.githubusercontent.com/69652463/197587534-c8ef8530-975b-45d9-9949-3541a8d20b4c.png)

**Comment by Mart-User** [comment]
Language: Brazilian
Where: Lobby -> Weapon selection
Key/Value translation:
"[english]asw_weapon_sentry_cannon" "Cannon Sentry Gun Case"
"asw_weaponl_sentry_cannon" "Canhão Sentinela de Alta Velocidade FAI"

Current text: Canhão Sentinela de Alta Velocidade...
Expected text: Canhão Sentinela de Alta Velocidade FAI

Possible solution: widen the container of the label + fix alignment text to the left

![asw_weaponl_sentry_cannon_ptbr](https://user-images.githubusercontent.com/4331009/201133859-6102cc19-4a8a-4dc2-abf8-dabed7686e90.png)

**Reply by Duke-M-commits** [reply]
Additionally I recommend leaving in-universe abbreviations like in the english version, there are multiple "IAF" textures on weapon and equipment boxes, I also think IAF is mentioned in some soundfiles (which are only available in english). Such inconsistencies are easier to recognize than the fact that "Interstellar Armed Forces" isn't "IAF" in most translations.

**Reply by BenLubar** [reply]
Does this also happen on the weapon selection menu on the beta branch? It's being completely redesigned.

**Reply by Mart-User** [reply]
@BenLubar in beta it displays without issues. Thanks!

Preview:
![image](https://user-images.githubusercontent.com/4331009/201494146-cd58bda7-0b78-4e07-a407-8a3ab35d910e.png)

**Reply by Duke-M-commits** [reply]
It's back? @BenLubar 
![image](https://github.com/user-attachments/assets/eafa7866-fac4-4e8e-ad9e-1df94338c259)

**Comment by Duke-M-commits** [comment]
Language: German
Where: Inventory -> Swarmopedia
Possible solution: widen (multiline) the container of the label

![image](https://user-images.githubusercontent.com/69652463/201522065-660af9d8-2ecf-4373-ba06-77b203d2d41c.png)

**Comment by Duke-M-commits** [comment]
Language: German
Where: Lobby -> Mission Description Labels
![image](https://user-images.githubusercontent.com/69652463/218036189-a747b973-14d9-44ec-8a1e-84e0c415bf77.png)

**Comment by Duke-M-commits** [comment]
Language: English
Where: Workshop -> Add-ons
![image](https://user-images.githubusercontent.com/69652463/236682903-fbe6c6d3-249a-4c63-a3ea-5ff7f49ab553.png)

**Comment by Duke-M-commits** [comment]
Where: Options -> Advanced Settings -> HUD

![image](https://github.com/ReactiveDrop/reactivedrop_translations/assets/69652463/195b6b5b-cd99-4604-9887-a69eac594c33)

**Comment by BenLubar** [comment]
The good news is that the ones that aren't fixed yet in this thread so far are on screens that are being redesigned this year. The bad news is that I have to actually do that.

**Comment by Duke-M-commits** [comment]
Language: English
Where: Options -> Options 1
Possible solution: widen (a bit) the container of the label
![image](https://github.com/ReactiveDrop/reactivedrop_translations/assets/69652463/7dcb0c11-0348-4d8f-a1e5-e4e143c6883e)

Also, I made the background a bit brighter to show how the boxes could use a bit more height too:
![image](https://github.com/ReactiveDrop/reactivedrop_translations/assets/69652463/2aafa4ba-56cf-49a2-9e37-cfefffb3fe79)
